api/scripts/run-docker.sh

```diff
@@ -60,6 +63,8 @@ elif [ "$1" == "dump-organisation-to-s3" ]; then
     dump_organisation_to_s3 "$2" "$3" "$4"
 elif [ "$1" == "dump-organisation-to-local-fs" ]; then
     dump_organisation_to_local_fs "$2" "$3"
+elif [ "$1" == "migrate_analytics_db" ]; then
```

Comment:
Bit of Sophie's choice here! I'd rather all of these were consistent when it comes to '_' or '-' but we've used '_' for migrate_identities... 🤔 

How difficult would it be to change from migrate_identities to migrate-identities ? It's only our SaaS platform that uses it, right, so it should be easy enough.

Reply:
Yep, done